From: Hans Wippel <hwippel@linux.vnet.ibm.com>
Subject: qeth: VNIC Characteristics support
Patch-mainline: never (kmsg is SLES only)
References: bsc#1061024 FATE#323301

Summary:     qeth: VNIC Characteristics support
Description: Add support for VNIC Characteristics (VNICC). VNICCs are features
             of HiperSockets that define how packets are handled by the
             underlying network hardware. For example, if the VNICC flooding is
             configured, ethernet frames to unknown destination MAC addresses
             are received.

Upstream-Description:

             s390/kmsg: add VNIC Characteristics msg documentation

             This patch adds the documentation of VNIC Characteristics output
             to kmsg.

             Signed-off-by: Hans Wippel <hwippel@linux.vnet.ibm.com>


Signed-off-by: Hans Wippel <hwippel@linux.vnet.ibm.com>
Signed-off-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 Documentation/kmsg/s390/qeth |   17 +++++++++++++++++
 1 file changed, 17 insertions(+)

--- a/Documentation/kmsg/s390/qeth
+++ b/Documentation/kmsg/s390/qeth
@@ -934,6 +934,23 @@
  */
 
 /*?
+ * Text: "%s: Configuring the VNIC characteristics failed\n"
+ * Severity: Error
+ * Parameter:
+ *   @1: bus ID of the qeth device
+ * Description:
+ * A qeth device was set online or subject to device recovery. Not all virtual
+ * network interface card (VNIC) characteristics for this device could be
+ * configured. The device might not support VNIC characteristics or might
+ * support only a subset of the possible VNIC characteristics. The state of the
+ * sysfs attributes reflects the current configuration of VNIC characteristics
+ * on the device.
+ * User action:
+ * Read the sysfs attributes in the vnicc subdirectory of the device to find out
+ * the current settings.
+ */
+
+/*?
  * Text: "%s: The qeth driver ran out of channel command buffers\n"
  * Severity: Warning
  * Parameter:
